### PR TITLE
[143]Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by Nelyne**
Annotation 2020-12-07 135314

### CODE REVIEW

**Review by Ireti203 — CHANGES_REQUESTED** [review]
Remove the spaces in your image names. You already have a product page, you don't have to create another one.

**Comment by Nelyne** [comment]
> Remove the spaces in your image names. You already have a product page, you don't have to create another one.

I think you are looking at the wrong codes, there aren't any images in the new PR I posted. The previous PR I submitted had images and I closed that PR and created a new one which are the initial codes you'll find and not the ones beneath. Kindly recheck, thank you

File: submissions/nkeirukaEnendu/HTML-Forms/index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+       <div>
+           <ul>
+               <li> <a href="#product">Gallery</a></li>
+               <li> <a href="#">FAQs</a></li>
+               <li> <a href="#contactUs">Contact Us</a></li>
+           </ul>
+       </div>
+   </nav>
+   <div id="product">
+       <img src="./img - Copy/Rectangle 11.png" alt="">
```
Comment (Ireti203) [inline submissions/nkeirukaEnendu/HTML-Forms/index.html:22]: The image name here Rectangle 11.png is not properly written. Remove the spaces in the image name. Also, your product page task has been merged earlier, you dont have to repeat it in this new Pull request.

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Review by adaugochi — CHANGES_REQUESTED** [review]
Remove spaces from your images names

**Comment by adaugochi** [comment]
Kindly update your PR title to the new task title that is on the app. Going forward, I will have to close this PR so you can then create a new one

**Pull request closed without merging**